src/modules/Core/Wallets/api.js

```diff
@@ -45,6 +56,26 @@ export const getReceiveAddress = (wallet: EdgeCurrencyWallet, currencyCode: stri
   return wallet.getReceiveAddress ? wallet.getReceiveAddress({ currencyCode }) : Promise.resolve(dummyEdgeReceiveAddress)
 }
 
+export const makeSpendInfo = (paymentProtocolInfo: EdgePaymentProtocolInfo): Promise<EdgeSpendInfo> => {
```

Comment:
This shouldn't return a Promise, it should just return the object:
{
   networkFeeOption: 'high',
   metadata: {
     name,
      notes
   },
   spendTargets
}

Reply:
This is a function that I created in an attempt to make core related stuff async